Sources/HttpPipeline/Format.swift

```diff
@@ -1,19 +0,0 @@
-public enum Format {
```

Comment:
I got rid of this in favor of `MediaType`. it seems that `MediaType` includes more things than what is necessary for `Content-Type` header, but there is still a ton of overlap. think it's worth finding a way to use it for both?